roles/benchmarking_deploy_coco_dataset/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Ensure that a hostname was passed in parameter
+  fail: msg="'benchmarking_node_hostname' must be provided"
+  when: benchmarking_node_hostname | default('', true) | length == 0
+
+- name: Ensure that a valid hostname name was passed in parameter
+  command: oc get nodes -l kubernetes.io/hostname={{ benchmarking_node_hostname }} -oname
+
+- name: Delete the mirror secret, if it exists
+  command:
+    oc delete secret/mirror-secret
+       -n {{ benchmarking_namespace }}
+       --ignore-not-found=true
```

Comment:
`--ignore-not-found` is true by default, no need for `--ignore-not-found=true`
Similarly below.

Reply:
oh you mean `--ignore-not-found=true` is like `test == true`?
true indeed :) but there are 46 places where it should be updated :#
will be for another PR!